pages/api/globalSearch.ts

```diff
@@ -44,19 +40,21 @@ export default async function handler(req: Request) {
       limit: 6,
     }
 
-  if (COLLECTION_SET_ID) {
-    queryParams.collectionsSetId = COLLECTION_SET_ID
-  } else if (COMMUNITY) {
-    queryParams.community = COMMUNITY
-  }
-
   supportedChains.forEach(async (chain) => {
-    const { reservoirBaseUrl, apiKey } = chain
+    const { reservoirBaseUrl, apiKey, collectionSetId, community } = chain
     const headers = {
       headers: {
         'x-api-key': apiKey || '',
       },
     }
+
+    const query = { ...queryParams }
+    if (collectionSetId) {
+      query.collectionsSetId = collectionSetId
+    } else if (community) {
+      query.community = community
+    }
+
     promises.push(
       fetcher(`${reservoirBaseUrl}/search/collections/v1`, queryParams, headers)
```

Comment:
I think we need to pass in `query` here instead of `queryParams`

Reply:
nice catch!